packages/service-registry-api/src/msg.rs

```diff
@@ -99,6 +99,12 @@ pub enum QueryMsg {
         chain_name: Option<ChainName>,
     },
 
+    #[returns(ServiceParamsOverride)]
```

Comment:
I'd prefer to return an Option instead of failing the query if there is no override